.github/workflows/daily.yml

```
@@ -19,6 +19,45 @@ on:
         default: 'master'
 
 jobs:
+  tests-with-elle:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install build dependencies
+        run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen
```

```suggestion
        run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen default-jdk
```
I guess we'll need jdk here.

---

yes, but made headless and to be consistent with above comments